schema/contact-list-contacts.yml

```diff
@@ -0,0 +1,21 @@
+contact_list_contacts:
+  list_name:
+    - string
+    - required
+    - name of the contact list
+    - example: "my_list"
+  list_id:
+    - string|integer
+    - required
+    - external id of the contact list
+    - example: "12345"
+  contact_id_type:
+    - string
+    - required
+    - type of the contact id
+    - example: "MDM_INFLUENCER | MDM_OUTLET | GORKANA_INFLUENCER | GORKANA_OUTLET | NGC3_INFLUENCER | NGC3_OUTLET | C3_INFLUENCER | C3_OUTLET"
```

Comment:
so I believe c3 will have 
private outlet
private contact
public contact
public outlet   

which type will be mapped to whom from these ?

Reply:
You're correct that the MDM_* values will be used for public outlets and public contacts. My assumption is that each of the other services will map to the Private versions (eg NGC3_INFLUENCER => private_contact).

I have no personal knowledge of how any of these alternate systems store these values, so I wouldn't be able to comment how to find the relevant data.

Reply:
@mjainrock4 , @sieglerz outlet and contact information is being gathered using below API
 "whitepages_url":"https://api.cision.com"
 End points: /v2/influencers/id  and /v2/influencer-lists?size={size}
 
 Mayank, i see you have committed some changes around this code in Upgrade service. please add any other information you may have.